frontend/src/styles/components/Team.scss

```
@@ -0,0 +1,143 @@
+.Team {
+  padding-top: 229px;
+  padding-bottom: 380px;
+  @include to(rem(1600)) {
+    padding: 140px 0;
+  }
+  @include to($lg) {
+    padding-bottom: 250px;
+  }
+  @include to($sm) {
+    padding-top: 113px;
+    padding-bottom: 185px;
+  }
+  .slick-slide {
+    transition: 0.5s;
+    outline: none;
+  }
+  .slick-active,
+  .slick-current {
+    img {
+      transform: scale(0.58);
+      margin-left: 20%;
+      @include to($lg) {
+        // margin-left: 0;
+        // transform: scale(0.95);
+      }
+      @include to($sm) {
+        // margin-left: 100px;
+      }
+    }
+  }
+  &__container {
+    width: 513px;
+    flex-shrink: 0;
+    @include to($lg) {
+      width: 100%;
+    }
+  }
+  &__inner {
+    margin-top: 80px;
+    @include to(rem(1600)) {
+      margin-top: 40px;
+    }
+    @include to($lg) {
+      display: block;
+      margin-top: 42px;
+    }
+    @include to($sm) {
+      margin-top: 32px;
+    }
+  }
+  &__about {
+    margin-top: -22px;
+    @include to(rem(1600)) {
+      margin-top: 0;
+    }
+  }
+  &__quote {
+    font-family: $font-medium;
+    font-size: rem(60);
+    letter-spacing: 1.2px;
+    line-height: rem(66);
+    text-transform: uppercase;
+    @include to(rem(1600)) {
+      font-size: rem(44);
+      line-height: 1.1;
+    }
+    @include to($lg) {
+      font-size: rem(48);
+      letter-spacing: 1.36px;
+      line-height: rem(52);
+    }
+    @include to($sm) {
+      font-size: rem(30);
+      line-height: 1.3;
+    }
+  }
+  &__descr {
```

Для улучшения читаемости, лучше использовать полные имена

```suggestion
  &__description {
```